b-real-time-htb.js

```diff
@@ -401,8 +425,8 @@ function BRealTimeHtb(configs) {
                 pmid: 'ix_brt_dealid'
             },
             lineItemType: Constants.LineItemTypes.ID_AND_SIZE,
-            callbackType: Partner.CallbackTypes.ID, // Callback type, please refer to the readme for details
-            architecture: Partner.Architectures.SRA, // Request architecture, please refer to the readme for details
+            callbackType: Partner.CallbackTypes.NONE, // Callback type, please refer to the readme for details
+            architecture: Partner.Architectures.FSRA, // Request architecture, please refer to the readme for details
             requestType: Partner.RequestTypes.ANY // Request type, jsonp, ajax, or any.
         };
         /* ---------------------------------------------------------------------------------------*/
```

Comment:
Can you please change the value of the `inputCentMultiplier` to 100 in 
https://github.com/indexexchange/b-real-time-adapter/blob/83305a3ec0833d93de91db670cab2b78b08d4aa7/b-real-time-htb.js#L452
and 
https://github.com/indexexchange/b-real-time-adapter/blob/83305a3ec0833d93de91db670cab2b78b08d4aa7/b-real-time-htb.js#L468